codeworld-base/src/Internal/CodeWorld.hs

```diff
@@ -68,7 +68,8 @@ simulationOf (initial, step, draw) = do
     rs <- chooseRandoms
     CW.simulationOf (initial rs)
                     (\dt w -> step (w, fromDouble dt))
-                    (toCWPic . draw) `catch` reportError
+                    (toCWPic . draw)
+                    `catch` reportError
```

Comment:
Sorry, I should have been clearer.  I'm trying to avoid the catch looking like a parameter, so let's please indent less than the parameters.  Four spaces past the do block indent would be consistent.